worlds/mmbn3/Rom.py

```diff
@@ -0,0 +1,276 @@
+from ndspy import lz10
+from Patch import APDeltaPatch
+
+import Utils
+import os
+import hashlib
+import bsdiff4
+
+from .BN3RomUtils import *  # We are literally going to use every single function from this
```

Comment:
Even if you use everything from a module, it is still recommended to not use `import *` because it makes it more difficult for linters and type checkers to check **how** you're using them.